.github/workflows/bundle_webapp_and_release_v3.yml

```diff
@@ -0,0 +1,183 @@
+name: Bundle Webapp and Release
+
+on:
+  push:
+    branches:
+      - 'future3/main'
+      - 'future3/develop'
+
+jobs:
+
+  check:
+    if: ${{ github.repository_owner == 'MiczFlor' }}
+    runs-on: ubuntu-latest
+
+    outputs:
+      tag_name: ${{ steps.vars.outputs.tag_name }}
+      release_type: ${{ steps.vars.outputs.release_type }}
+      check_abort: ${{ steps.vars.outputs.check_abort }}
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set Output vars
+        id: vars
+        env:
+          BRANCH_NAME: ${{ github.ref_name }}
+        run: |
+          # Official SemVer Regex definition
+          # https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string
+          # Needed changes to the regex:
+          # - '?:' capture command needed to be removed as it wasn't working in shell
+          # - '\d' had to be replaced with [0-9]
+          #
+          # Release versions like 1.0.0, 3.5.0, 100.4.50000+metadata
+          REGEX_VERSION_RELEASE="^(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)(\+([0-9a-zA-Z-]+(\.[0-9a-zA-Z-]+)*))?$"
+          #
+          # Prerelease versions like 1.0.0-alpha, 3.5.0-whatsoever.12, 100.4.50000-identifier.12+metadata
+          REGEX_VERSION_PRERELEASE="^(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)(-((0|[1-9][0-9]*|[0-9]*[a-zA-Z-][0-9a-zA-Z-]*)(\.(0|[1-9][0-9]*|[0-9]*[a-zA-Z-][0-9a-zA-Z-]*))*))?(\+([0-9a-zA-Z-]+(\.[0-9a-zA-Z-]+)*))?$"
+
+
+          # Get the version and calculate release type
+          VERSION=$(python ./src/jukebox/jukebox/version.py)
+          if echo "$VERSION" | grep -qoE "$REGEX_VERSION_RELEASE" ; then
+            RELEASE_TYPE=release
+          elif echo "$VERSION" | grep -qoE "$REGEX_VERSION_PRERELEASE" ; then
+            RELEASE_TYPE=prerelease
+          else
+            RELEASE_TYPE=none
+          fi
+
+          if [ "$BRANCH_NAME" == 'future3/main' -a "$RELEASE_TYPE" == 'release' ] || [ "$BRANCH_NAME" == 'future3/develop' -a "$RELEASE_TYPE" == 'prerelease' ] ; then
```

Comment:
Release type on develop maybe “nightly”?

Reply:
Its not excatly a 'nightly' as its not time triggered, so could be confusing.
And 'prerelease' is the  Semantic Versioning definition and also show on the GitHub Release.

Reply:
I was just wondering, if it’s a prerelease when it’s happening on every merge to future3/develop

Reply:
hmm ok i think i get your point. I would say yes and no ^^ 
In regards of the version, where the releasetype is calculated from and used, its a prerelease.
In regards of the source code, its closer to a nightly, even though we dont create new versions and tag.

But its just a string and only internally used in the action. So whatever fits best for all :D